tests/ui/test_ui_dashboard.py

```
@@ -1095,3 +1095,245 @@ async def test_multiple_refreshes(self, page_with_js_coverage: Page) -> None:
+        csv_button = page_with_js_coverage.locator('.download-btn[data-section="topRepositories"][data-format="csv"]')
+
+        # Listen for download event
+        try:
+            async with page_with_js_coverage.expect_download(timeout=5000) as download_info:
+                await csv_button.click()
+            download = await download_info.value
+
+            # Verify download properties
+            assert download.suggested_filename.endswith(".csv")
+            assert "github_metrics_" in download.suggested_filename
+        except Exception:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)